### ISSUE TITLE
[a11y][WCAG 3.2.1]: Screen reader focus is not moving to Filters results region after applying filters

### ISSUE THREAD

**Issue by dkatzz** [civiform/civiform#9915]
Observed Results:
--
After applying filters in "Programs" screen, screen reader focus is not moving to Filters output region and screen reader remains silent.

Screencast: Chromebook + Chromevox[here]( https://drive.google.com/file/d/1I9OW1f_02TkRtq0yfi7a4qbo3Q_T0OLa/view?usp=drive_link ) \
Screencast: Windows + NVDA[here](https://github.com/user-attachments/assets/1b4a29ba-4400-4679-8948-2a1d03d8a2d5)

User Impact:
-------
Users who rely on assistive technologies like screen reader will face difficulty if focus is not moving to output screen after activating any control

Expected Results:
--
After selecting any checkbox under "Available programs and services" region and hit "Apply selections" button, focus should move to the filters results region and should announce as " Programs based on your selections(1)"

Steps to Reproduce:
-------
0. Enable Chromevox using Ctrl+Alt+Z
1. Launch the URL[here]( https://ns.civiform.dev/programs )
2. Navigate to Checkbox controls at "Available programs and services - filter by category" region.
3. Select any checkbox among (Economic, General and Utilities) and hit "Apply selections" button
4. Now observe the screen reader verbalization after activating filter selections button

WCAG Details:
--
[WCAG 3.2.1][On-focus]( https://www.w3.org/WAI/WCAG22/quickref/?versions=2.0#on-focus )

Additional Information
--
N/A

Environment Details:
--
Machine: chromebook 13.3/Windows \
Browser: Version 132.0.6834.208 (Official Build) (64-bit) \
Screenreader: Chromevox/NVDA 2024.4.2135031 \
Bug Filling Time: 15 mins.
